libs/stokesian_dynamics/multi_array.hpp

```diff
@@ -0,0 +1,248 @@
+#pragma once
+
```

Comment:
Please add a high-level comment explaining what this header provides.

Reply:
Also the copyright header is missing. The `#pragma once` should be replaced by an ifdef. The documentation can be written as an `@file`-style doxygen comment, e.g.
```c++
/*
 * Copyright (C) 2019-2020 The ESPResSo project
 * ...
 */
/** @file
 *  Paragraph describing the file
 */

#ifndef SD_MULTI_ARRAY_HPP
#define SD_MULTI_ARRAY_HPP
...
#endif
```